test/e2e/config/e2e_conf.yaml

```diff
@@ -62,16 +62,12 @@ providers:
     type: "url"
     contract: v1beta1
     files:
-    - sourcePath: "${PWD}/metadata.yaml"
+    - sourcePath: "../../../metadata.yaml"
```

Comment:
This metadata.yaml file that e2e uses is different from the one in the root directory. Please revert this change

Reply:
if you mean reverting `${PWD}/metadata.yaml` it will end up using the same file